packages/zoe/src/typeGuards.js

```diff
@@ -321,10 +324,10 @@ export const ZoeStorageManagerIKit = harden({
 });
 
 export const ZoeServiceI = M.interface('ZoeService', {
-  install: M.call(M.any()).returns(M.promise()),
-  installBundleID: M.call(M.string()).returns(M.promise()),
+  install: M.call(M.any()).optional(M.string()).returns(M.promise()),
+  installBundleID: M.call(M.string()).optional(M.string()).returns(M.promise()),
   startInstance: M.call(M.eref(InstallationShape))
-    .optional(IssuerPKeywordRecordShape, M.any(), M.any())
+    .optional(IssuerPKeywordRecordShape, M.any(), M.any(), M.string())
```

Comment:
I'm pretty sure `terms` and `privateArgs` must be records.
```suggestion
    .optional(IssuerPKeywordRecordShape, M.record(), M.record(), M.string())
```

Reply:
Done